cli/src/program_v4.rs

```diff
@@ -111,52 +108,44 @@ impl ProgramV4SubCommands for App<'_, '_> {
                 .setting(AppSettings::SubcommandRequiredElseHelp)
                 .subcommand(
                     SubCommand::with_name("deploy")
-                        .about("Deploy a program")
+                        .about("Deploy a new or redeploy an existing program")
                         .arg(
-                            Arg::with_name("program_location")
+                            Arg::with_name("program-location")
```

Comment:
Would this be optional for redeployment scenario? i.e. in case a buffer account is being used.

Reply:
You mean the source could not just be a file but also a different (buffer) account?

Reply:
Yes, in the old flow, for redeployment we didn't provide a source file.

Reply:
Oh ok. Somehow I misread the code then.